src/containers/NFT/NFTHeader/Settings.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react'
+import './styles.scss'
+import { useTranslation } from 'react-i18next'
+
+interface Props {
+  flags: string[]
+}
+
+const Settings = ({ flags }: Props) => {
+  const { t } = useTranslation()
+
+  const burnable = flags.includes('lsfBurnable') ? 'enabled' : 'disabled'
+  const onlyXRP = flags.includes('lsfOnlyXRP') ? 'enabled' : 'disabled'
+  const trustLine = flags.includes('lsfTrustLine') ? 'enabled' : 'disabled'
```

Comment:
need to name this better potentially to describe what this flag does. see comment above.